[core]/es_extended/server/main.lua

```diff
@@ -378,7 +378,7 @@ AddEventHandler('playerDropped', function(reason)
 	if xPlayer then
 		TriggerEvent('esx:playerDropped', playerId, reason)
         local job = xPlayer.getJob().name
-        ESX.JobsPlayerCount[job] = (ESX.JobsPlayerCount[job] or 1) -1
+        ESX.JobsPlayerCount[job] = ((ESX.JobsPlayerCount[job] and ESX.JobsPlayerCount[job] > 0) and ESX.JobsPlayerCount[job] or 1) -1
```

Comment:
It's okey, but please have a variable for it because you use it more than once.

For example
local currentJobCount = ESX.JobsPlayerCount[job]
ESX.JobsPlayerCount[job] = ((currentJobCount and currentJobCount > 0) and currentJobCount or 1) -1